src/core/components/parameter-row.jsx

```diff
@@ -251,15 +251,15 @@ export default class ParameterRow extends Component {
     if (isOAS3) {
       schema = this.composeJsonSchema(schema)
     }
-    
+
     let format = schema ? schema.get("format") : null
     let isFormData = inType === "formData"
     let isFormDataSupported = "FormData" in win
     let required = param.get("required")
 
-    const typeLabel = fn.jsonSchema202012.getType(immutableToJS(schema))
-    const type = fn.jsonSchema202012.foldType(immutableToJS(schema?.get("type"))) 
-    const itemType = fn.jsonSchema202012.foldType(immutableToJS(schema?.getIn(["items", "type"]))) 
+    const typeLabel = fn.getSchemaObjectType(immutableToJS(schema))
```

Comment:
This looks a bit confusing. For getting a `typeLabel` we're using `getSchemaObjectType`. For actual `type` we're using `getSchemaObjectTypeLabel`. Shouldn't it be reversed?